app/views/shared/_footer_nav.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="footer row">
```

Comment:
This 'footer' class will then need to change as well

Reply:
done